pkgs/by-name/as/astro-language-server/package.nix

```diff
@@ -0,0 +1,97 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, stdenvNoCC
+, jq
+, moreutils
+, nodePackages
+, cacert
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Recursion should be avoided by replacing it with `finalAttrs`.

Reply:
Fixed this